test/code_size/hello_world_wasm2js.js

```diff
@@ -15,6 +15,7 @@ function e(b) {
             for (var v, p = 0, t = a, w = f.length, y = a + (3 * w >> 2) - ("=" == f[w - 2]) - ("=" == f[w - 1]); p < w; p += 4) a = m[f.charCodeAt(p + 1)], 
             v = m[f.charCodeAt(p + 2)], c[t++] = m[f.charCodeAt(p)] << 2 | a >> 4, t < y && (c[t++] = a << 4 | v >> 2), 
             t < y && (c[t++] = v << 6 | m[f.charCodeAt(p + 3)]);
+            return c;
```

Comment:
Any idea what this is? Looks odd.

Reply:
No, that does look odd. Hard to know why without knowing which function this actually is.

Reply:
It's plausible that this is some function that in some way uses bulk memory.

Reply:
Hmm, based on the `charCodeAt` and `<< 6` etc., this looks like `base64Decode`:

https://github.com/emscripten-core/emscripten/blob/4cbcb262fa917fade59d291d2c85514c7a798c17/src/base64Decode.js#L38-L58

That doesn't use bulk memory AFAICT. Strange that it changed.

Reply:
Anyhow, it must be a weird closure quirk, as `x()` is called once and the return value isn't used.